test/unit/spec/UtilBrowserFeatures_spec.js

```diff
@@ -0,0 +1,20 @@
+import BrowserFeatures from 'util/BrowserFeatures';
+
+describe('util/BrowserFeatures', () => {
+  describe('getUserLanguages', () => {
+    it('returns a non-read-only array on Chrome/Firefox', () => {
+      jest.spyOn(navigator, 'languages', 'get').mockImplementation(() => Object.freeze(['en_US', 'en']));
+      let errorThrown = false;
+      let result = null;
+      try {
+        result = BrowserFeatures.getUserLanguages();
+        result[0] = 'foo'
+      }
+      catch (err) {
+        errorThrown = true;
+      }
+      expect(errorThrown).toBe(false);
```

Comment:
how about test the happy path instead, like property of the return value can be changed, which describes the expected behaviour.

Reply:
https://github.com/okta/okta-signin-widget/pull/2544/files/efa8d43212bfdd965bdd6d3ba3d1179e907c0e19#diff-4f8e33ace31dfd3c70ac41d51bc10cccb6a149623fae160bd11dcd7a9dfdd199R11

This does modify a field of the return value - aka happy path

Reply:
I mean try ... catch may not be needed, test fails if error is thrown.